tests/unit/test_exceptions.py

```diff
@@ -21,17 +22,59 @@ def test_str(self):
         assert str(PRAWException("foo")) == "foo"
 
 
-class TestAPIException:
-    def test_inheritance(self):
-        assert isinstance(APIException(None, None, None), PRAWException)
+class TestRedditErrorItem:
+    def test_equality(self):
+        resp = ["BAD_SOMETHING", "invalid something", "some_field"]
+        error = RedditErrorItem(*resp)
+        error2 = RedditErrorItem(*resp)
+        assert error == error2
+        assert error != 0
+
+    def test_property(self):
+        error = RedditErrorItem(
+            *["BAD_SOMETHING", "invalid something", "some_field"]
+        )
+        assert (
+            error.error_message
+            == "BAD_SOMETHING: 'invalid something' on field 'some_field'"
+        )
 
     def test_str(self):
-        exception = APIException(
-            "BAD_SOMETHING", "invalid something", "some_field"
+        error = RedditErrorItem(
+            *["BAD_SOMETHING", "invalid something", "some_field"]
+        )
+        assert (
+            str(error)
+            == "BAD_SOMETHING: 'invalid something' on field 'some_field'"
+        )
+
+    def test_repr(self):
+        error = RedditErrorItem(
+            *["BAD_SOMETHING", "invalid something", "some_field"]
+        )
+        assert (
+            repr(error) == "RedditErrorItem(error_type=BAD_SOMETHING, message="
+            "invalid something, field=some_field)"
+        )
+
+
+class TestRedditAPIException:
+    def test_inheritance(self):
+        assert isinstance(
+            RedditAPIException([[None, None, None]]), PRAWException
         )
```

Comment:
```suggestion
        assert issubclass(RedditAPIException, PRAWException)
```

Reply:
This change should really be propagated to the other subclass checks.